xml/System/Console.xml

```diff
@@ -4995,8 +5003,9 @@ This method can be used to reacquire the standard output stream after it has bee
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- If value is `null`, nothing is written to the standard output stream.
+ The output is encoded according to <xref:System.Console.OutputEncoding%2A>.
```

Comment:
(I reordered this to put my comment first, because otherwise there's a syntactic ambiguity about whether my comment is conditioned on the "value is `null`" from the other comment.)